src/components/AboutApp.js

```diff
@@ -0,0 +1,34 @@
+import { h } from 'preact'
+import { useI18n, useSoftkey } from 'hooks'
+
+export const AboutApp = () => {
+  const i18n = useI18n()
+  const APP_VERSION = JSON.stringify(require('../../package.json').version).slice(1, -1)
+
+  useSoftkey('AboutApp', {
+    right: i18n.i18n('softkey-read-more'),
+    onKeyRight: () => window.open('https://wikimediafoundation.org/'),
+    left: i18n.i18n('softkey-close'),
+    onKeyLeft: () => history.back()
+  }, [])
+
+  return (
+    <div class='aboutapp'>
+      <div class='header'>{i18n.i18n('gallery-about-header')}</div>
```

Comment:
Reusing `gallery-about-header` although the name may be slightly confusing here

Reply:
I think it's better to have a different key here as they are different

Reply:
Yea I suppose it doesn't cost us anything to add another key with the same text, just slightly more redundancy I guess. What about renaming `gallery-about-header` to something like just `about-header`?

Reply:
`about-header` looks good =)

Reply:
Renaming is fine in this case. TWN can handle that without losing all the translations.